src/frontend/screens/Security/index.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+import { FormattedMessage, defineMessages } from "react-intl";
+
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ListDivider,
+} from "../../sharedComponents/List";
+import IconButton from "../../sharedComponents/IconButton";
+import { BackIcon, SaveIcon } from "../../sharedComponents/icons";
+import HeaderTitle from "../../sharedComponents/HeaderTitle";
+import { SecurityContext } from "./SecurityContext";
+import { useNavigation } from "react-navigation-hooks";
+import SettingsContext from "../../context/SettingsContext";
+
+const m = defineMessages({
+  title: {
+    id: "screens.Security.title",
+    defaultMessage: "Security",
+  },
+  securitySubheader: {
+    id: "screens.Security.securitySubheader",
+    defaultMessage: "Device Security",
+  },
+  passcodeHeader: {
+    id: "screens.Security.passcodeHeader",
+    defaultMessage: "App Passcode",
+  },
+  passDesriptionPassNotSet: {
+    id: "screens.Security.passDesriptionPassNotSet",
+    defaultMessage: "Passcode not set",
+  },
+  passDesriptionPassSet: {
+    id: "screens.Security.passDesriptionPassSet",
+    defaultMessage: "Passcode is set",
+  },
+  killPasscodeHeader: {
+    id: "screens.Security.killPasscodeHeader",
+    defaultMessage: "Kill Passcode",
+  },
+  killPassDescriptonPassNotSet: {
+    id: "screens.Security.killPassDescriptonPassNotSet",
+    defaultMessage: "To use, enable App Passcode",
+  },
+  killPassDescriptonPassSet: {
+    id: "screens.Security.killPassDescriptonPassSet",
+    defaultMessage: "Protect your device against seizure",
+  },
+  deviceBackup: {
+    id: "screens.Security.deviceBackup",
+    defaultMessage: "Device Backup",
+  },
+  paperKey: {
+    id: "screens.Security.paperKey",
+    defaultMessage: "Paper Key",
+  },
+  paperKeyDes: {
+    id: "screens.Security.paperKeyDes",
+    defaultMessage: "Reinstate your account if this device is lost",
+  },
+});
+
+export const Security: NavigationStackScreenComponent = () => {
+  const { passIsSet } = React.useContext(SecurityContext);
+  const [{ experiments }] = React.useContext(SettingsContext);
+  const { navigate } = useNavigation();
+
+  React.useEffect(() => {
+    if (!experiments.appPasscode) navigate("Settings");
+  }, []);
+
+  const [passCodeDes, killPassCodeDes] = React.useMemo(
+    () =>
+      passIsSet
+        ? [m.passDesriptionPassSet, m.killPassDescriptonPassSet]
+        : [m.passDesriptionPassNotSet, m.killPassDescriptonPassNotSet],
+    [passIsSet]
+  );
```

Comment:
couple of nits:

- is the memoization necessary here? guessing it may not be but if there's a noticeable behavior it prevents, fine to keep
- probably would make sense to extract the arrays as constants defined outside of the component e.g.

```ts
// Outside of the Security component
const PW_DESCS_WHEN_SET = [m.passDesriptionPassSet, m.killPassDescriptonPassSet]
const PW_DESCS_WHEN_NOT_SET = [m.passDesriptionPassNotSet, m.killPassDescriptonPassNotSet]

export const Security = () => {
  // ... 
  const [passCodeDes, killPassCodeDes] = React.useMemo(
    () =>
      passIsSet
        ? PASSWORD_DESCRIPTION_SET
        : PASSWORD_DESCRIPTION_NOT_SET
    [passIsSet]
  );
}
```